### ISSUE TITLE
provide a example

### ISSUE THREAD

**Issue by kidwm** [ByteMuse/cssnextify#2]
is there any further example about how to use this tool ?

**Comment by ByteMuse** [comment]
Sorry for the dalay. Examples are inside `README.md` file.

**Comment by kidwm** [comment]
@ByteMuse I mean a small and simple project example.

I can't figure out how to compile css file with the main.js in the `README.md` file.

**Comment by MoOx** [comment]
You probably need to require a css file. Should not be more complicated than that right @ByteMuse?

**Comment by ByteMuse** [comment]
Yes, css file must be required from `main.js` file. I will update the examples. Thank you.

**Comment by kidwm** [comment]
@MoOx Thank you for the instruction.
@ByteMuse Glad to see the update and close this issue.
